.github/workflows/build-release-qt6.yml

```diff
@@ -261,7 +261,7 @@ jobs:
     runs-on: windows-2019
     env:
       # Qt 6.5 windeployqt doesn't deploy DLLs!
-      QT_VERSION: 6.4.2
+      QT_VERSION: 6.6.1
```

Comment:
I tested the Windows version, there are some DLLs missing. The App doesn't start. I guess windeployqt is still not working correctly in Qt 6.6.1 😠

Reply:
Does it say which dlls?

Reply:
Qt6PrintSupport, Qt6Widgets, Qt6Gui, Qt6Sql, then it stops.

Reply:
I see all the needed dlls in the log

```
-a---          11/20/2023 12:18 PM        1765520 Qt6Core5Compatd.dll
-a---          11/17/2023  5:41 AM       16518800 Qt6Cored.dll
-a---          11/17/2023  5:41 AM       25592464 Qt6Guid.dll
-a---          11/17/2023  5:41 AM        4366480 Qt6Networkd.dll
-a---          11/17/2023  5:41 AM        4741264 Qt6OpenGLd.dll
-a---          11/17/2023  5:42 AM        1072272 Qt6PrintSupportd.dll
-a---          11/20/2023  6:14 AM       16275600 Qt6Qmld.dll
-a---          11/20/2023  6:13 AM        2558096 Qt6QmlModelsd.dll
-a---          11/20/2023  6:13 AM         248464 Qt6QmlWorkerScriptd.dll
-a---          11/20/2023  6:15 AM       18922128 Qt6Quickd.dll
-a---          11/17/2023  5:42 AM         816272 Qt6Sqld.dll
-a---          11/20/2023  5:35 AM        1125520 Qt6Svgd.dll
-a---          11/20/2023 12:05 PM         613520 Qt6WebSocketsd.dll
-a---          11/17/2023  5:42 AM       16437392 Qt6Widgetsd.dll
-a---          11/17/2023  5:42 AM         412304 Qt6Xmld.dll
```

They are debug dlls though. Release mode needs to be explicitly specified when running qmake I think.

Reply:
Hm, I see they are ending with `d` 🤔

Reply:
Its because of debug arg in `windeployqt --debug QOwnNotes.exe`. Probably should let windeployqt figure this out by itself. Also maybe `.` should be specified instead of QOwnNotes.exe so that it scans the whole build folder for dependencies

Reply:
🤷🏻